pwa/src/apiService/resources/filterCount.ts

```diff
@@ -0,0 +1,24 @@
+import { TSendFunction } from "../apiService";
+import { AxiosInstance } from "axios";
+
+export default class FilterCount {

```

Comment:
Why is this called `..Count`? We're not actually doing anything with the count, nor are we only returning the count. Also, in the files which implement this, we're not using the count either.

Reply:
I think this was implemented to actually _only_ show categories that have entries (so with a count), maybe naming this (e.g.):

```
useAvailableFilters();

getCategories();

// implementation could be useAvailableFilters().getCategories();
```